index.html

```diff
@@ -10,38 +10,12 @@
 				<a class="nav" href="#portfolio">Portfolio</a>
 				<a class="nav" href="#contact">Contact information</a>		
 			</header>
-			
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			<p>AHHHHHHH</p>
-			
-			
+		
+			<section>
+				<p class="intro">HEELLL</p>
+				<p class="intro">OOOOO</p>
+				<p class="little-intro">I am Josh</p>		
+		</section>
 			
 			
 			
```

Comment:
Closing html tag and head tag on top is not closed

Reply:
Thankyou